src/mapDataManipulation/mapDataManipulation.ts

```diff
@@ -0,0 +1,56 @@
+// manipulatePixel is a function that transforms pixel's (R,G,B) values into new (R,G,B) values
+export async function mapDataManipulation(originalBlob: Blob, manipulatePixel: any): Promise<Blob> {
+  let imgObjectUrl: any;
+  try {
+    const imgCanvas = document.createElement('canvas');
```

Comment:
I think we should use a unique name for the `imgCanvas` instead of `'canvas'` - since the function is a Promise it might be run in parallel resulting in a race condition that overwrites the previous canvas before `mapDataManipulation` finishes?

Reply:
If you are referring to the `canvas` inside of `document.createElement()`, that's the `tagName` of the HTML element, not the `id` / `name` / `class`. We could create a [`HTMLUnknownElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLUnknownElement), but that would be unnecessary work in my opinion. ([MDN docs for `document.createElement()`](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement))

Reply:
You are correct. I assumed that this was the reference and not the `tagName`. Thanks for the link